source/administration/image-proxy.rst

```diff
@@ -3,24 +3,21 @@
 Image Proxy
 ================================
 
-Using image proxies means that clients make only secure requests. By allowing image requests to go straight to third-party
-servers, tracking pixels is allowed. Users can put invisible images in posts that logs time and location data
-for every user that views the post. An image proxy protects user privacy by eliminating this direct interaction with 
-third-party servers.
+Using an image proxy makes it so that all requests for images made by Mattermost clients will go through the proxy instead of contacting third-party servers directly. This helps protect user privacy by preventing third-party servers from tracking who views an image. This also prevents the use of tracking pixels, invisible images that do the same thing without the user even seeing an image.
```

Comment:
```suggestion
Using an image proxy means that all requests for images made by Mattermost clients will go through the proxy instead of contacting third-party servers directly. This helps protect user privacy by preventing third-party servers from tracking who views an image. This also prevents the use of tracking pixels (invisible images that do the same thing without the user even seeing an image).
```